src/main/java/com/minecolonies/coremod/inventory/GuiItemHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.minecolonies.coremod.inventory;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.TextComponentTranslation;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+import java.awt.*;
+
+@SideOnly(Side.CLIENT)
+public class GuiItemHandler extends GuiContainer
+{
+    private static final ResourceLocation TEXTURE = new ResourceLocation("textures/gui/container/generic_54.png");
+    private final IInteractiveItemHandler handler;
+    private final int inventoryRows;
+
+    public GuiItemHandler(IInteractiveItemHandler handler)
+    {
+        super(new ContainerItemHandler(Minecraft.getMinecraft().thePlayer.inventory, handler));
+        this.handler = handler;
+        this.allowUserInput = false;
+        this.inventoryRows = handler.getSlots() / 9;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 9 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)